source/includes/_build.md

```diff
@@ -15,28 +21,45 @@ https://github.com/hyperledger/iroha /path/to/iroha
 
 ### How to run development environment
 
-``` shell
-/path/to/iroha/scripts/run-iroha-dev.sh
-```
+Run the script `run-iroha-dev.sh`, contained in the folder `scripts`: `sh .../iroha/scripts/run-iroha-dev.sh`
 
-You will be attached to interactive environment for development and testing with `iroha` folder mounted from host machine.
+After you execute this script, following things happen:
 
-Docker environment will be removed when you logout from the container.
+ 1. It checks if you don't have containers with Iroha already running. If yes — it will reattach you to interactive shell. Assuming if you don't have any, then:
+ 2. The script will download iroha-docker-develop, redis and postgres images. Iroha image contains all development dependencies, and is based on top of ubuntu:16.04.
+ 3. Three containers are created and launched.
+ 4. The user is attached to the interactive environment for development and testing with `iroha` folder mounted from the host machine.
 
-### Build `iroha` and run tests
+<aside class="notice">
+Docker environment will be removed when you detach from the container.
+</aside>
 
-Build:
-```
+
+### Build project and run tests
+
+> Build
+
+``` bash
 cmake -H. -Bbuild;
 cmake --build build -- -j$(nproc)
 ```
 
-`irohad` and `iroha-cli` binaries will be in `./build/bin` directory.
+<aside class="notice">
+On macOS $(nproc) variable does not work. Check number of logical cores with sysctl -n hw.ncpu
+</aside>
+
+We use CMake to build platform-dependent build files. Run shell commands from "Build" section on the right. Built binaries (`irohad` and `iroha-cli`) will be in `./build/bin` directory.
+
+After you built the project — please run tests to check the operability of the daemon.
 
 ### Add to irohad and iroha-cli to path (optional)
 
 export PATH=/path/to/iroha/build/bin:$PATH
 
+<aside class="notice">
```

Comment:
Same as above.

Reply:
It looks like note

Reply:
![screen shot 2017-12-16 at 16 31 59](https://user-images.githubusercontent.com/11841667/34070913-ae391c3e-e27e-11e7-84c8-019ac3ed5a3b.png)

Reply:
Oh, I see. I opened this page throw github which ignores such tags.